diabetes_regression/util/model_helper.py

```diff
@@ -44,37 +44,44 @@ def get_model_by_tag(
         # Validate params. cannot be None.
         if model_name is None:
             raise ValueError("model_name[:str] is required")
-        if tag_name is None:
-            raise ValueError("tag_name[:str] is required")
-        if tag_value is None:
-            raise ValueError("tag[:str] is required")
+
         if aml_workspace is None:
+            print("No workspace defined - using current experiment workspace.")
             aml_workspace = get_current_workspace()
 
-        # get model by tag.
-        model_list = AMLModel.list(
-            aml_workspace, name=model_name,
-            tags=[[tag_name, tag_value]], latest=True
-        )
+        model_list = None
+        tag_ext = ""
+
+        # Get lastest model
+        # True: by name and tags
+        if tag_name is not None or tag_value is not None:
```

Comment:
I believe both of them should be not None.
Or at least tag_name.

Reply:
Updated accordingly.